proto/migration/migration.go

```diff
@@ -93,3 +93,45 @@ func V1Alpha1AttSlashingToV1(v1alpha1Slashing *ethpb_alpha.AttesterSlashing) *et
 		Attestation_2: V1Alpha1IndexedAttToV1(v1alpha1Slashing.Attestation_2),
 	}
 }
+
```

Comment:
Some tests for these?

Reply:
Yeah, the `migration` package doesn't have any tests... Would it be OK if I create a separate PR with tests for the whole package?

Reply:
Sounds good 👍